### PR TITLE
Jarv/jenkins var updates for deploy

### PR DESCRIPTION
**Pull request by jarv**

### CODE REVIEW

File: playbooks/edx-east/edx_provision.yml [hunk 1]
```diff
@@ -19,8 +19,10 @@
...
   hosts: launched
   sudo: True
-  gather_facts: True
+  gather_facts: False
   roles:
     # gh_users hash must be passed
     # in as a -e variable
     - gh_users
+    # Sets hostname to edx-server (default)
+    - hostname
```
Comment (feanil) [inline playbooks/edx-east/edx_provision.yml:28]: Is this still necessary if you have to re-run rabbit anyway?

Comment (jarv) [inline playbooks/edx-east/edx_provision.yml:28]: Yeah it isn't, I'll do a quick test to make sure things still come up without it included.

File: playbooks/roles/rabbitmq/tasks/main.yml [hunk 2]
```diff
@@ -18,6 +18,9 @@
 - name: rabbitmq | stop rabbit cluster
   service: name=rabbitmq-server state=stopped
 
+- name: rabbitmq | send sigterm to any running rabbitmq processes
+  shell: killall beam rabbitmq-server epmd
```
Comment (feanil) [inline playbooks/roles/rabbitmq/tasks/main.yml]: Shouldn't this only happen on refresh?

Comment (jarv) [inline playbooks/roles/rabbitmq/tasks/main.yml]: I think this should run every time assuming that even if the service is
already stopped there may be lingering processes.

File: playbooks/edx-east/edx_provision.yml (hunk 1 above)
Comment (jarv) [inline playbooks/edx-east/edx_provision.yml:28]: Changing the hostname caused problems for rabbit even after clearing out the mnesia dir and restarting both rabbit and the app.  I am going to leave it in for now.

**Comment by feanil** [comment]
looks good.

**Pull request merged**